packages/plugin-react-native-unhandled-rejection/test/rejection-handler.test.ts

```diff
@@ -48,21 +52,21 @@ describe('plugin: react native rejection handler', () => {
 
     // the rejection tracker waits 100ms before reporting TypeError as unhandled
     // so be generous and wait 3x that
-    setTimeout(() => done(), 300)
+    setTimeout(done, 300)
   })
 
-  it('should be disabled when enabledErrorTypes.unhandledRejections=false', done => {
-    const c = new Client({
-      apiKey: 'api_key',
-      enabledErrorTypes: { unhandledRejections: false, unhandledExceptions: true }
-    })
-    c._setDelivery(client => ({
-      sendEvent: (payload) => {
+  it('should be disabled when autoDetectUnhandledRejections=false', (done) => {
+    expect.assertions(1)
+
+    const c = new Client({ apiKey: 'api_key', autoDetectUnhandledRejections: false })
```

Comment:
TypeScript and test failing due to `autoDetectUnhandledRejections` not being valid. Looks like this test did not get updated with the changes related to auto detection.